code/game/objects/items/weapons/twohanded.dm

```diff
@@ -272,3 +272,36 @@
 
 	force = 16
 	throwforce = 25
+
+/obj/item/binoculars
+	name = "binoculars"
+	desc = "Used for long-distance surveillance."
+	icon_state = "binoculars"
+	item_state = "syringe_kit"
```

Comment:
Just think it should be more obvious - if you walk into someone looking through 'nocs, it's not immediately obvious what they're doing. You would have to examine them, and even then, you can't tell if they are actually looking through the noculars. This could do with being its own sprite.